packages/nhsuk-frontend/src/nhsuk/components/panel/template.njk

```diff
@@ -1,11 +1,33 @@
 {% from "nhsuk/macros/attributes.njk" import nhsukAttributes %}
 
-{% set headingLevel = params.headingLevel if params.headingLevel else 1 -%}
+{%- set headingLevel = params.headingLevel if params.headingLevel else 1 -%}
 
-<div {%- if params.id %} id="{{ params.id }}"{% endif %} class="nhsuk-panel
-  {%- if params.classes %} {{ params.classes }}{% endif %}"
-  {{- nhsukAttributes(params.attributes) }}>
-  <h{{ headingLevel }} class="nhsuk-panel__title">
+{# Set classes for this component #}
```

Comment:
```suggestion
{#- Set classes for this component #}
```

Reply:
Resolved in https://github.com/nhsuk/nhsuk-frontend/pull/1196/commits/3a5dabb60f4d6cd83df5020f309a5a999ef35c77